dvc/path_info.py

```diff
@@ -253,8 +253,6 @@ def path(self):
 
 
 class HTTPURLInfo(URLInfo):
-    __hash__ = URLInfo.__hash__
```

Comment:
Not sure why this was there. Maybe, the `__hash__` are not compatible because of the below overrides?

Reply:
@skshetry Because we override `__eq__` below?

Reply:
Yup, figured it out. Thanks.